Makefile

```diff
@@ -333,8 +337,10 @@ deploy: namespace deploy-crds ## Deploy the operator from the manifests in the d
 .PHONY: deploy-local
 deploy-local: namespace image-to-cluster deploy-crds ## Deploy the operator from the manifests in the deploy/ directory and the images from a local build
 	$(SED) 's%$(IMAGE_REPO)/$(APP_NAME):latest%$(RELATED_IMAGE_OPERATOR_PATH)%' deploy/operator.yaml
+	$(SED) 's%$(IMAGE_REPO)/$(RELATED_IMAGE_OPENSCAP_NAME):$(RELATED_IMAGE_OPENSCAP_TAG)%$(RELATED_IMAGE_OPENSCAP_PATH)%' deploy/operator.yaml
```

Comment:
I think the tag referenced here will be incorrect, TAG is "latest" and RELATED_IMAGE_OPENSCAP_TAG is "1.3.4". So let's use RELATED_IMAGE_OPENSCAP_TAG all around.

Reply:
Just a note: We will need a push target for the openscap image to run before the next upstream release.

Reply:
I used tag, because in existing openscap_build, 
```
.PHONY: openscap-image
openscap-image:
	$(RUNTIME) build --no-cache -t $(RELATED_IMAGE_OPENSCAP_PATH):$(TAG) $(OPENSCAP_DOCKER_CONTEXT)
```